### DISCUSSION TITLE
LDAP 2.0.0

### DISCUSSION THREAD

**Discussion by rizlas** [netbox-community/netbox-docker#798] [Getting Help] [answered]
Hi,
in docker hub currently there is no tag for ldap. Is image with tag v3.2.6-2.0.0 using ldap?

Thank you

**Comment by tobiasge** [comment, marked as answer]
See the release notes: https://github.com/netbox-community/netbox-docker/releases/tag/2.0.0

**Comment by rizlas** [comment]
Oh sorry, I missed that yesterday. Thank you

**Comment by rizlas** [comment]
I've tested this new image but unfortunately LDAP is not working with the same specs prior to the update. 

`SERVER_DOWN({'result': -1, 'desc': "Can't contact LDAP server", 'ctrls': [], 'info': '(unknown error code)'})`

I can ping the ldap server both from host machine and container.
